### PR TITLE
[stu] Create waiting page

### PR DESCRIPTION
**Pull request by ariellam**
Adds the waiting page for students.

Screen Shot 2021-04-17 at 11 31 13 AM

Will rebase off of `main` once `login-page` has been merged!

### CODE REVIEW

File: src/student/WaitingPage.tsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  max-width: 18em;
+  flex: 1;
+  text-align: center;
+`;
+
+export const WaitingPage = () => {
+  return (
+    <Page>
+      <PageColumn>
+        <TitleText color="white">quizem</TitleText>
```
Comment (BenLanglois) [inline src/student/WaitingPage.tsx:35]: I created a `StudentPageBase` component in the `login-page` branch that could be used here.

**Review by BenLanglois — CHANGES_REQUESTED** [review]
Looks good. I added more to the `login-page` branch, so maybe set that branch as the target instead of `main`.

**Comment by BenLanglois** [comment]
> Looks good. I added more to the `login-page` branch, so maybe set that branch as the target instead of `main`.

That branch has been merged into main now, so don't worry about changing the base anymore.

**Comment by ariellam** [comment]
> > Looks good. I added more to the `login-page` branch, so maybe set that branch as the target instead of `main`.
> 
> That branch has been merged into main now, so don't worry about changing the base anymore.

Cool, I'll rebase from main!

File: src/student/WaitingPage.tsx (hunk 1 above)
Comment (alexgirard) [reply src/student/WaitingPage.tsx:35]: +1

**Review by BenLanglois — APPROVED** [review]

**Pull request merged**